prog/dftb+/lib_dftbplus/main.F90

```diff
@@ -522,6 +522,13 @@ subroutine processGeometry(env, iGeoStep, iLatGeoStep, tWriteRestart, tStopDrive
               & iGeoStep, tStopScc, eigvecsReal, reks)
         end if
 
+      #:if WITH_TRANSPORT
+        ! Overrides input charges with uploaded contact charges
+        if (tUpload) then
+          call overrideContactCharges(qInput, chargeUp, transpar, qBlockIn, blockUp)
+        end if
+      #:endif
+
```

Comment:
Currently, REKS is not compatible with transport calculation, so this part have to be removed.